app/code/Magento/Sitemap/Controller/Adminhtml/Sitemap/Save.php

```diff
@@ -5,12 +5,74 @@
  */
 namespace Magento\Sitemap\Controller\Adminhtml\Sitemap;
 
-use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpPostActionInterface;
 use Magento\Framework\App\Filesystem\DirectoryList;
 use Magento\Framework\Controller;
+use Magento\Framework\Validator\StringLength;
+use Magento\MediaStorage\Model\File\Validator\AvailablePath;
+use Magento\Sitemap\Model\SitemapFactory;
 
-class Save extends \Magento\Sitemap\Controller\Adminhtml\Sitemap
+/**
+ * Save sitemap controller.
+ */
+class Save extends \Magento\Sitemap\Controller\Adminhtml\Sitemap implements HttpPostActionInterface
```

Comment:
Hi @irajneeshgupta. I believe HttpPostActionInterface is not the case here since this interface was introduced within a scope of 2.3.

Reply:
@rogyar , 
I will modify it .
Thanks.